tools/windows-node-installer/pkg/types/types.go

```diff
@@ -231,7 +231,19 @@ func (w *Windows) GetSSHClient() error {
 		HostKeyCallback: ssh.InsecureIgnoreHostKey(),
 	}
 
-	sshClient, err := ssh.Dial("tcp", w.Credentials.GetIPAddress()+":22", config)
+	var err error
+	var sshClient *ssh.Client
+	// Retry if we are unable to create a client as the VM could still be executing the steps in its user data. We
+	// cannot reuse the entries in the retry package as they are too granular.
+	for retries := 0; retries < 5; retries++ {
```

Comment:
Does this provide us enough safety moving forward? Windows is slow to boot... And Env dependent. I would retry every min for 10 min to try and cover other use cases like VMware in the future.

Reply:
I am fine increasing the time to 10 minutes. We will have to do the same in WMCO.